agent/engine/docker_task_engine.go

```diff
@@ -1140,36 +1142,11 @@ func (engine *DockerTaskEngine) AddTask(task *apitask.Task) {
 		})
 		task.SetKnownStatus(apitaskstatus.TaskStopped)
 		task.SetDesiredStatus(apitaskstatus.TaskStopped)
-		engine.emitTaskEvent(task, err.Error())
+		engine.EmitTaskEvent(task, err.Error())
 		return
 	}
 	if task.IsEBSTaskAttachEnabled() {
-		if csiTask, ok := engine.loadedDaemonTasks["ebs-csi-driver"]; ok {
-			logger.Info("engine ebs CSI driver is running", logger.Fields{
-				field.TaskID: csiTask.GetID(),
-			})
-		} else {
-			// TODO update 'ebs-csi-driver' as config param or const
-			var csiStartErr error
-			if ebsCsiDaemonManager, ok := engine.daemonManagers["ebs-csi-driver"]; ok {
-				csiTask, csiStartErr = ebsCsiDaemonManager.CreateDaemonTask()
-				engine.loadedDaemonTasks["ebs-csi-driver"] = csiTask
-				engine.AddTask(csiTask)
-				logger.Info("docker_task_engine: Added EBS CSI task to engine")
-			} else {
-				csiStartErr = errors.New("Unable to find ebs-csi-driver in engine")
-			}
-			if csiStartErr != nil {
-				logger.Error("Unable to start ebsCsiDaemon for task in the engine", logger.Fields{
-					field.TaskID: task.GetID(),
-					field.Error:  err,
-				})
-				task.SetKnownStatus(apitaskstatus.TaskStopped)
-				task.SetDesiredStatus(apitaskstatus.TaskStopped)
-				engine.emitTaskEvent(task, err.Error())
-				return
-			}
-		}
+		logger.Info("Task is EBS TaskAttachEnabled")
```

Comment:
I think we should add the task ARN to this log otherwise it's not clear which task is EBS enabled.